Sources/NIOSSL/NIOSSLHandler.swift

```diff
@@ -190,7 +199,53 @@ public class NIOSSLHandler : ChannelInboundHandler, ChannelOutboundHandler, Remo
         self.doFlushReadData(context: context, receiveBuffer: receiveBuffer, readOnEmptyBuffer: true)
         self.writeDataToNetwork(context: context, promise: nil)
     }
-    
+
+    // TODO: how do I test this?
```

Comment:
With `EmbeddedChannel`, you can fire the input closed event into the pipeline manually.